crates/nu-command/src/platform/input/input_listen.rs

```diff
@@ -69,7 +69,13 @@ There are 4 `key_type` variants:
     media - dedicated media keys (play, pause, tracknext ...)
     other - keys not falling under previous categories (up, down, backspace, enter ...)"#
     }
-
+    fn examples(&self) -> Vec<Example> {
+        vec![Example {
+            description: "Listen for a keyboard shortcut and find out how nu receives it",
+            example: "input list --types [key]",
```

Comment:
`input list` is a valid but completely different command :wink: 

```suggestion
            example: "input listen --types [key]",
```

Reply:
i feel you, if it's too hot where you are :relieved: 
i had the same two weeks ago :scream: